test/mlir/conversion/onnx_to_torch/flatten/flatten_axis_three.onnx.mlir

```diff
@@ -0,0 +1,11 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<2x3x4x5xf32>) -> tensor<24x5xf32> attributes {input_names = ["0"], output_names = ["1"]} {
+  //CHECK: %[[START:.*]] = torch.constant.int 0
+  //CHECK: %[[END:.*]] = torch.constant.int 2
+  //CHECK: torch.aten.flatten.using_ints %arg0, %[[START]],  %[[END]] :
```

Comment:
This test doesn't check much, I would have expected it to check both generated `flatten` ops and also include the types in the assertions

I edited the other test (axis 2) to show what I mean. It's failing because of the thing about types I mentioned elsewhere